python/onnx_converter/onnxconverter.py

```diff
@@ -0,0 +1,302 @@
+# Oggn
```

Comment:
Turn this into the way that the other converter is written with a class and a main function